src/stream.c

```diff
@@ -388,6 +388,10 @@ static void rtp_handler(const struct sa *src, const struct rtp_header *hdr,
 		s->rx.pseq = hdr->seq - 1;
 		flush = true;
 	}
+	else if (hdr->m && s->type == MEDIA_AUDIO) {
+		/* The marker bit indicates the beginning of a talkspurt. */
+		flush = true;
+	}
```

Comment:
For the jbuf:
- in case of SSRC change, there is already a `jbuf_flush()`.
- in case of timestamp/seq number jump --> needs a fix in `jbuf.c`

Reply:
jbuf will not run empty because `jbuf_put()` and `jbuf_get()` is called in same thread. The timestamp/seq number jump can not be defined clearly.
Thus added a timeout check in this `re` PR: https://github.com/baresip/re/pull/370